README.md

```diff
@@ -0,0 +1,47 @@
+# ci-operator
+
+ci-operator automates and simplifies the process of building and testing
+OpenShift component images (i.e. any `openshift/origin-{component}` images).
+
+Given a Git repository reference and a component-specific JSON configuration
```

Comment:
don't call-out JSON as we will change that in the future

Reply:
IMO, once we do that, we can fix that in the doc. Now it's telling the truth. I will adjust the wording though to inform about the future.